package.json

```diff
@@ -242,6 +250,11 @@
                 }
             ],
             "view/item/context": [
+                {
+                    "command": "azureFunctions.selectSubscriptions",
+                    "when": "view == azureFunctionsExplorer && viewItem == azureextensionui.azureSubscription",
+                    "group": "inline"
+                },
```

Comment:
Make sure this doesn't show up in the command palette:
```
"commandPalette": [
    {
        "command": "azureFunctions.selectSubscriptions",
        "when": "never"
    }
]
```

Reply:
done